main/api/ws.js

```diff
@@ -39,6 +39,10 @@ const handler = (socket, req) => {
       }
     }
     payload._origin = origin
+    
+    if (origin) {
```

Comment:
can origin ever be undefined here?

Reply:
I do get spammed with instances of this being undefined when I run frame locally, will remove the conditional if I can find out why...

Reply:
weird, not getting these undefined origins any more

Reply:
removing for now